packages/components/src/theme/ComposerTheme/Components/Badge.ts

```diff
@@ -2,10 +2,10 @@ import { ComponentStyleConfig } from '@chakra-ui/react';
 
 export const statusStyles = {
   custom: {
-    backgroundColor: 'primary-default',
-    borderColor: 'primary-default',
-    colorOutlineSubtle: 'text-badge-outline',
-    colorSolid: 'text-badge-solid',
+    backgroundColor: 'background-badge-custom',
+    borderColor: 'background-badge-custom',
```

Comment:
Is this the new design? Do they really want to use the same color for border and background?

Reply:
Yes